bin/tools/tools.ml

```diff
@@ -20,7 +22,9 @@ module Which = struct
   let info = Cmd.info ~doc "which"
 
   let group =
-    Cmd.group info (List.map [ Ocamlformat; Ocamllsp ] ~f:Tools_common.which_command)
+    Cmd.group
+      info
+      (List.map [ Ocamlformat; Ocamllsp; Utop ] ~f:Tools_common.which_command)
```

Comment:
I assume these will always get updated together. If so perhaps the sub-commands should be defined in a single list?

Reply:
Ah, also, for my edification, why isn't this `Dev_tool.all`?

Reply:
Ah good idea. This file has had many refactors over the past few months. I'll update it to use `Dev_tool.all`.